src/routers/bookmark_router.py

```diff
@@ -0,0 +1,123 @@
+from fastapi import APIRouter, Depends, HTTPException
+from typing import List, Union
+from pydantic import BaseModel, validator
+from sqlmodel import Session, select, Field
+
+from authentication import KeycloakUser, get_user_or_raise
+from database.session import get_session
+from database.model.bookmark.bookmark import Bookmark
+from http import HTTPStatus
+from database.model.concept.concept import AIoDConcept
+from database.model.helper_functions import non_abstract_subclasses
+from datetime import datetime
+from routers.helper_functions import get_asset_type_by_abbreviation
+
+
+class BookmarkRead(BaseModel):
+    resource_identifier: str = Field(description="The identifier of the resource being bookmarked.")
+    created_at: datetime = Field(description="The time when the bookmark was created in ISO 8601 format.")
+
+    class Config:
+        json_encoders = {datetime: lambda dt: dt.isoformat()}
+
+
+def create(url_prefix: str = "") -> APIRouter:
+    router = APIRouter()
+
+    for path in [
+        f"{url_prefix}/v2/bookmarks",
+        f"{url_prefix}/bookmarks",
+    ]:
+
+        @router.get(
+            path,
+            tags=["User"],
+            description="Return all assets you have bookmarked.",
```

Comment:
```suggestion
            description="Return all your bookmarks.",
```
Probably better to indicate the assets themselves are not returned?